src/shogun/clustering/GMM.cpp

```diff
@@ -335,6 +339,11 @@ float64_t CGMM::train_smem(int32_t max_iter, int32_t max_cand, float64_t min_cov
 		}
 		if (!better_found)
 			break;
+
+		this->observe<float64_t>(iter, "log_likelihood", "Log Likelihood", cur_likelihood);
+		this->observe<SGVector<float64_t>>(iter, "coefficients");
+		this->observe<std::vector<CGaussian*>>(iter, "gaussians", "Gaussians Components",m_components);
```

Comment:
Isn’t this a registered parameter? I think it’s not that nice that he description is here even though this belongs to the class.
It should be like he coefficients no?

Reply:
Yes, indeed. It is fixed now.